app/dashboard/EditProfile/EditProfile.tsx

```diff
@@ -0,0 +1,395 @@
+        </View>
+      </View>
+
+      <View className="flex-row justify-between mt-4">
+        <TouchableOpacity
+          className={`${bgColor} px-4 py-2 border-b-4 ${
+            activeTab === 'EDITING PROFILE'
+              ? 'border-indigo-400 shadow-sm rounded-lg'
+              : 'border-transparent shadow-sm rounded-lg'
+          }`}
+          onPress={() => handleTabPress('EDITING PROFILE')}
+        >
+          <Text className={`${textColor}`}>EDITING PROFILE</Text>
+        </TouchableOpacity>
+        <TouchableOpacity
+          className={`${bgColor} px-4 py-2 border-b-4 ${
+            activeTab === 'UPLOAD RESUME'
+              ? 'border-indigo-400 shadow-sm rounded-lg'
+              : 'border-transparent shadow-sm rounded-lg'
+          }`}
+          onPress={() => handleTabPress('UPLOAD RESUME')}
+        >
+          <Text className={`${textColor}`}>UPLOAD RESUME</Text>
+        </TouchableOpacity>
+      </View>
+
+      {tab === 0 && (
+        <View className={`bg-[#272728] px-4 pt-10 pb-6 rounded-lg ${bgColor} shadow-lg`}>
+          <TouchableOpacity
+            onPress={() => router.push('/dashboard/trainee/Profile')}
+            className="flex-row items-center mb-4 "
+          >
+            <Text className="text-white text-lg bg-action-500 p-2 rounded-lg">
+              ← Back to Profile
+            </Text>
+          </TouchableOpacity>
+
+          <View className="pb-4">
+            <Text className={`${textColor} pb-2`}>First Name</Text>
+            <View
+              className={`flex-row items-center ${bgColor} rounded-lg shadow border-2 border-[#D2D2D2]`}
+            >
+              <TextInput
+                testID="first-name"
+                className={`p-3 rounded-lg ${textColor}`}
+                placeholder="First Name"
+                placeholderTextColor="#888"
+                onChangeText={formik.handleChange('firstName')}
+                value={formik.values.firstName}
+              />
+            </View>
+
+            <Text className="text-error-500 my-1">{formik.errors.firstName}</Text>
+
+            <Text className={`${textColor} pb-2`}>Last Name</Text>
+            <View
+              className={`flex-row items-center ${bgColor} rounded-lg shadow border-2 border-[#D2D2D2]`}
+            >
+              <TextInput
+                testID="last-name"
+                className={`p-3 rounded-lg ${textColor}`}
+                placeholder="Last Name"
+                placeholderTextColor="#888"
+                onChangeText={formik.handleChange('lastName')}
+                value={formik.values.lastName}
+              />
+            </View>
+            <Text className="text-error-500 my-1">{formik.errors.lastName}</Text>
+
+            <Text className={`${textColor} pb-2`}>Telephone</Text>
+            <View
+              className={`flex-row items-center ${bgColor} rounded-lg shadow border-2 border-[#D2D2D2]`}
+            >
+              <TextInput
```

Comment:
when i update the phone number there not being updated ???

Reply:
**Done**